### DISCUSSION TITLE
Difference in indicator values compared to TradingView - like STC. etc?

### DISCUSSION THREAD

**Discussion by FSquant** [DaveSkender/Stock.Indicators#1287] [Q&A]
Dear all,

one quick question:
why is that there are some indicators in this lib which show massivly different values than if i compare them with patforms like TradingView?

Example:

the STC in Skender.NET gets massivly different results (with same config) than on trading view: On some days, Skender.NE says a value around 14.xx, while on TradingView the value is 50+.

Did someone else find this curious?

**Comment by DaveSkender** [comment]
Here's a large list of reasons.  In short, the problem is rarely with this library.
- #801

With that said, I've not seen (or reviewed) the STC difference.
I do like to double-check, since we're not entirely immune from bugs or issues.

**Reply by DaveSkender** [reply]
Send me a link to what you're seeing with STC and I'll take a look when I have a free moment.

**Comment by FSquant** [comment]
Good Evening, Dave!

Wooohooo... what a blazing fast answer - and what a good one!
In fact, i can confirm esp one sentence from this post: "in general, TV is not a good source of truth" :-D
Im using several data providers and checking back on TV does not match in a lot of cases, thanks for confirming - i was already scratching my head for a very long time, also their support does not answer at all on those questions. (somewhen i showed them the primary data feed with clearly different data than on their platform, they didnt react at all)

So - OK - understood, that means:
We can take the computations from your - GREAT - library as "the true one" like the "industry standard computation"

Thanks a lot for the response, so we can skip the whole comparison shit with TV!

Regards

**Reply by DaveSkender** [reply]
Hehe, fair enough, but I'd still defer to original authors' mathematical formulas for sources of truth.  We only aim to be true to their original math whenever possible.  Thanks for checking in!

**Comment by FSquant** [comment]
OK,
so that means you are slashing old text-books at the local math-library?
Very cool :-)

So actually your library saved me at least 1 year of development time, i'd guess: Or maybe i wouldnt have used all that stuff, if i would have had to implement it on my own. Thanks a lot for your stuff!
Let me ask: Are there any "commercial" or "corporate" users of your lirbary or is just a bunch of finance guys on the internet?

**Reply by DaveSkender** [reply]
Yes!  There are commercial organizations using the library.  It's hard to tell exactly how many since I can't actually track who's downloading it.  With that said, if you look at the [used by](https://github.com/DaveSkender/Stock.Indicators/network/dependents) log of public repos, you'll see a lot of personal bot-traders -- so the vast majority are likely individuals -- not sure.
